.travis.yml

```diff
@@ -7,3 +7,4 @@ script:
 
 after_success:
   - bash <(curl -s https://codecov.io/bash)
+dist: trusty
```

Comment:
What's the purpose of this change?

Reply:
I ran into this issue: https://travis-ci.com/microsoft/spring-cloud-azure/builds/120492482 with the build. I chatted with @sophiaso who found https://travis-ci.community/t/error-installing-oraclejdk8-expected-feature-release-number-in-range-of-9-to-14-but-got-8/3766/4 and https://travis-ci.community/t/oracle-jdk-11-and-10-are-pre-installed-not-the-openjdk-builds/785/16. It is thought that the distribution was selected randomly if not configured explicitly. https://docs.travis-ci.com/user/reference/trusty/.